Sources/CardPayments/Models/Card.swift

```diff
@@ -66,10 +66,26 @@ public struct Card: Encodable {
 
 struct Attributes: Codable {
 
+    let customer: Customer?
+    let vault: Vault?
     let verification: Verification
 }
 
 struct Verification: Codable {
 
     let method: String
 }
+
+struct Customer: Codable {
+    
+    let id: String
+}
+
+enum StoreInVault: String, Codable {
+    case onSuccess = "ON_SUCCESS"
+}
+
+struct Vault: Codable {
+    
+    let storeInVault: StoreInVault
```

Comment:
Since this will always be `ON_SUCCESS` do we need an enum to store this string? Why not just `let storeInVault = "ON_SUCCESS"`?

Are we planning to add more cases?

Reply:
I just heard that there can be other cases but I am not sure if we are handling it at this stage. 
If there are, we'd have to add a field in the CardRequest.

Reply:
If there's values we _know_ we will add soon, I understand preparing for it. But we try to follow YAGNI and not add things we don't need, since it often muddies up the code.

Reply:
That property is in "ServiceCard" now.